vistopia/main.py

```diff
@@ -130,19 +131,33 @@ def save_show(ctx, **argv):
 @main.command("save-transcript")
 @click.option("--id", type=click.INT, required=True)
 @click.option("--episode-id", help="Episode ID in the form '1-3,4,8'")
+@click.option("--single-file-exec-path", type=click.Path(),
+              help="Path to the single-file CLI tool")
+@click.option("--cookie-file-path", type=click.Path(),
+              help="Path to the browser cookie file")
```

Comment:
Please mention in the help that it is used in combination with the single-file option only.